<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppDialog</class>
 <widget class="QDialog" name="AppDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Web App</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>209</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>490</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Manifest URL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Profiles:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Keywords:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Categories:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbBoxCategories">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbBoxKeywords">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="appIcon">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>61</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEditName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>28</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>Website name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Icon:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEditManifest">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>28</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>https://www.website.com/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEditDescription">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>28</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbBoxProfiles">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblNote">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Note: It is strongly recommended to install each web app into a separate profile due to a hard-to-fix bug.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
